Some make command warnings

There’s this:
```
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.1.2
```

And that:
```
/nextcloud/apps/contacts/js/services/contact_service.js
  247:56  warning  Don't make functions within a loop  no-loop-func

✖ 1 problem (0 errors, 1 warning)
```

Just some cosmetics probably but would be good to fix? :)

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/49557596-some-make-command-warnings?utm_campaign=plugin&utm_content=tracker%2F46751899&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F46751899&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/js/services/contact_service.js
+++ b/js/services/contact_service.js
@@ -249,6 +249,7 @@ angular.module('contactsApp')
 				num++;
 				continue;
 			}
+			// eslint-disable-next-line no-loop-func
 			this.create(newContact, addressBook, '', true).then(function(xhrContact) {
 				if (xhrContact !== false) {
 					var xhrContactName = xhrContact.displayName();

